docs/services/mosquitto.md

```diff
@@ -0,0 +1,23 @@
+# Mosquitto Ansible Role
+
+
+[Mosquitto](https://mosquitto.org/) is an open source [MQTT](https://en.wikipedia.org/wiki/MQTT) broker.
+
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+mosquitto_enabled: true
+```
+
+## Usage
+
+After installation, you can use `just run-tags mosquitto-add-user --extra-vars=username=<username> --extra-vars=password=<password>"` to create a user. For the setting to take effect, you must restart the container.
```

Comment:
Seems like there's an unmatched `"` here.

"Restart the container" is not very helpful to more novice users. We'd better tell people how: `just restart-group mosquitto`